lib/podlet.js

```diff
@@ -130,6 +131,27 @@ const PodiumPodlet = class PodiumPodlet {
             writable: true,
         });
 
+        // Skip a tick to ensure the metric stream has been consumed
+        setTimeout(() => {
+            const moduleVersion = pkg.version;
+            const segments = moduleVersion.split('.').map(value => {
+                return parseInt(value, 10);
+            });
+
+            this.metrics.metric({
+                name: 'podium_podlet_version_info',
+                description: '@podium/podlet version info',
+                labels: [
+                    { name: 'version', value: moduleVersion },
+                    { name: 'major', value: segments[0] },
+                    { name: 'minor', value: segments[1] },
+                    { name: 'patch', value: segments[2] },
+                ],
+                value: 1,
+                type: 1,
+            });
+        });
```

Comment:
`setTimeout()` should have a timeout value. It will never be 0. Though; what is the purpose here? Is the purpose to execute this on the next tick of the eventloop or actually do a timeout after x milliseconds? If running on next tick is the puspose we should use `setImmediate()` instead of `setTimeout()`.

Reply:
switched to `setImmediate()`